From: =?UTF-8?q?Rafa=C5=82=20Mi=C5=82ecki?= <rafal@milecki.pl>
Date: Fri, 15 Dec 2023 10:17:21 +0100
Subject: [PATCH] list: don't backport list_count_nodes()
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

It's redundant in OpenWrt as it backports it on its own. This fixes:
backport-include/linux/list.h:11:22: error: redefinition of 'list_count_nodes'

Signed-off-by: Rafał Miłecki <rafal@milecki.pl>
---
 backport-include/linux/list.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/backport-include/linux/list.h
+++ b/backport-include/linux/list.h
@@ -3,7 +3,7 @@
 #include_next <linux/list.h>
 #include <linux/version.h>
 
-#if LINUX_VERSION_IS_LESS(6,3,0)
+#if 0 /* OpenWrt backports list_count_nodes() on its own */
 /**
  * list_count_nodes - count nodes in the list
  * @head:	the head for your list.
